bitnami/spark/templates/statefulset-master.yaml

```diff
@@ -149,7 +163,12 @@ spec:
           configMap:
             name: {{ .Values.master.configurationConfigMap }}
         {{- end }}
-        {{- if .Values.security.certificatesSecretName }}
+        {{- if .Values.metrics.enabled }}
+        - name: metricsconfig
+          configMap:
+            name: "metrics-config"
+        {{- end }}
+       {{- if .Values.security.certificatesSecretName }}
```

Comment:
Please, align this line with an extra space.

Reply:
Pinging because it's not yet done